app/scripts/controllers/dashboard.coffee

```diff
@@ -544,4 +544,22 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', [
         $scope.anchorScroll = () ->
             $anchorScroll()
 
+        $scope.showModal = (total) ->
```

Comment:
oh right, yeah I guess `schoollist`, not `schoollistmodal`. I think it makes more sense there.